bucket/png2html.json

```diff
@@ -1,5 +1,5 @@
 {
-    "homepage": "https://www.engr.mun.ca/~holden/png2html.html",
+    "homepage": "http://www.engr.mun.ca/~holden/png2html.html/",
```

Comment:
`.html/` doesn’t look right, as directories usually don’t end with file extensions. Please confirm or correct.

Reply:
Removed the forward slash, all good now.